src/pipelines/html.rs

```diff
@@ -109,9 +109,9 @@ impl HtmlPipeline {
 
         // Assemble a new output index.html file.
         let output_html = target_html.html(); // TODO: prettify this output.
-        fs::write(format!("{}/index.html", self.cfg.dist.display()), output_html.as_bytes())
+        fs::write(self.cfg.dist.join("index.html"), output_html.as_bytes())
             .await
-            .context("error writing finalized HTML output")?;
+            .context("error writing finalized HTML output")?; // TODO: show also error details (in a verbose mode?)
```

Comment:
This actually does surface the underlying error when it makes it up to the top-level of the app. It will print the context message first, and then will print out the chain of errors leading up to the context message.

Reply:
Unless you had something else in mind, we can probably remove that `TODO`.